weatherman/weatherobj.py

```diff
@@ -0,0 +1,66 @@
+"""
+Weather object for keeping relevant details
+Provides keys for  data in dictionary
+"""
+from datetime import datetime
+from utils import is_hotter_day
+
+
+class WeatherObj:
```

Comment:
Class name should not have Object in them. Why not create one WeatherData for one year and use it like so.
```
{
  '1995': YearlyWeatherSummary(),
  ...
}
```